app/models/encryptable.rb

```diff
@@ -27,6 +28,9 @@ class Encryptable < ApplicationRecord
   validates :name, presence: true
   validates :description, length: { maximum: 4000 }
 
+  validate :assert_human_receiver?, if: :transferred?
```

Comment:
Currently, there is an error with this validation. It's not called with the test, but it's working in rails console